packages/richtext-lexical/src/features/indent/client/index.tsx

```diff
@@ -80,17 +87,31 @@ const toolbarGroups: ToolbarGroup[] = [
   ]),
 ]
 
-export const IndentFeatureClient = createClientFeature({
-  plugins: [
-    {
-      Component: IndentPlugin,
-      position: 'normal',
+export const IndentFeatureClient = createClientFeature<IndentFeatureProps>(({ props }) => {
+  const disabledNodes = props.disabledNodes ?? []
+  return {
+    plugins: [
+      {
+        Component: () => IndentPlugin({ clientProps: props }),
```

Comment:
After returning `sanitizedClientFeatureProps`, this should be unnecessary - IndentPlugin will automatically receive the client props